config/seatunnel-env.sh

```diff
@@ -19,4 +19,7 @@
 # Home directory of spark distribution.
 SPARK_HOME=${SPARK_HOME:-/opt/spark}
 # Home directory of flink distribution.
-FLINK_HOME=${FLINK_HOME:-/opt/flink}
\ No newline at end of file
+FLINK_HOME=${FLINK_HOME:-/opt/flink}
+
+# Control the ascii logo string, set to empty to disable it
+export SEATUNNEL_ASCII_LOGO=SeaTunnel
```

Comment:
hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

Reply:
what's your suggestion?

Reply:
We may never change the logo, so it should be a final constant.

Reply:
> hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

+1

Reply:
The main purpose is hide the logo print, change the logo word is just a fun for some body who want to custom.

Reply:
> hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

+1

Reply:
I agree with @leo65535 , hard code with the logo is better. But I do not have strong opinions here.